api/swagger-spec/oapi-v1.json

```diff
@@ -17700,6 +17700,10 @@
      "customStrategy": {
       "$ref": "v1.CustomBuildStrategy",
       "description": "CustomStrategy holds the parameters to the Custom build strategy"
+     },
+     "jenkinsPipelineStrategy": {
```

Comment:
1) a known issue; once this PR is merged and there's an origin release we can update the fabric8 console to do the right thing
https://github.com/fabric8io/fabric8-console/issues/24
2) the Jenkins stuff should appear on the fabric8 page once we've got a release out of origin with this fix; as we can then add it to the schema

3 & 4) what @jimmidyson said :) I thought for 4) that the existing App Dashboard with pipelines & environment views were gonna be reused in the OpenShift console?